src/Peering/Peering/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+## Version 0.1.1
```

Comment:
@rileymckenna you can remove the `## Version 0.1.1` header and keep everything under `## Upcoming Release` -- we will do the versioning